src/context/SelectedBooking.js

```diff
@@ -4,11 +4,14 @@
 import * as React from 'react';
 
 type Props = {|
+  // isOpen required here only to trigger processUrl when opening/closing SmartFAQ
+  isOpen: boolean, // eslint-disable-line react/no-unused-prop-types
   children: React.Node,
 |};
 type StateValues = {
   selectedBooking: ?number,
   bookingPage: 'SINGLE_BOOKING' | 'ALL_BOOKINGS',
+  userSelectedBooking: boolean,
```

Comment:
Why is this required? Isn't `selectedBooking` `null` when and only when `userSelectedBooking` is `false`?

Reply:
I put there explanation into a comment, to keep our initial behaviour, you need to differentiate when specific booking was set automatically by url and when user specifically went to list of another bookings and selected one - in such case I don't want to change his settings

Reply:
:+1: